fullstop-plugins/fullstop-registry-plugin/src/main/java/org/zalando/stups/fullstop/plugin/RegistryPlugin.java

```diff
@@ -131,19 +146,25 @@ private void validateSourceWithKio(final CloudTrailEvent event, final String app
 
         if (!source.equals(artifact)) {
             violationSink.put(
-                    new ViolationBuilder(
-                            format("Application: %s has not a valid artifact for version: %s.", applicationId,
-                                    applicationVersion)).withEventId(getCloudTrailEventId(event)).withRegion(
-                            getCloudTrailEventRegion(event)).withAccoundId(getCloudTrailEventAccountId(event)).build());
+                new ViolationBuilder(
+                    format("Application: %s has not a valid artifact for version: %s.", applicationId,
+                        applicationVersion)).withEventId(getCloudTrailEventId(event)).withRegion(
+                    getCloudTrailEventRegion(event)).withAccoundId(getCloudTrailEventAccountId(event)).build());
+        }
+
+        Map<String, String> tags = Maps.newHashMap();
+        try {
+
+            tags = this.pieroneOperations.listTags(team, applicationId);
+        } catch (HttpClientErrorException e) {
+            LOG.error("Could not get the tags for team {} and applicationId {}", team, applicationId, e);
```

Comment:
log level `warn` would imho be more appropriate, because the method flow is not interrupted here. `error` means, that something bad happened and the app is not able to recover from it.